docs/integrate/dbeaver/index.md

```diff
@@ -0,0 +1,90 @@
+(dbeaver)=
+# DBeaver
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/DBeaver_logo.svg/512px-DBeaver_logo.svg.png){w=120px}](https://dbeaver.io/)
+```
+
+[DBeaver] is a multipurpose cross-platform database tool for developers,
+database administrators, analysts, and everyone working with data.
+
+It is available as an open-source version _DBeaver Community_ and
+as a commercial version _DBeaver PRO_.
+
+
+## Install
+
+::::{grid} 2
+:::{grid-item}
+For connecting to CrateDB, the [CrateDB JDBC Driver] will be used.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/ebd11a53-4d4a-4a9b-bed2-25909e618929){w=480px}
+:::
+::::
+
+
+## Connect
+
+::::{grid} 2
+
+:::{grid-item}
+Please specify database URL and credentials of your CrateDB cluster.
+
+When connecting to [CrateDB Self-Managed] on localhost,
+for evaluation purposes, use:
+```
+jdbc:crate://localhost:5432/
+```
+
+When connecting to [CrateDB Cloud], use:
+```
+jdbc:crate://<clustername>.cratedb.net:5432/
+```
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/3288ab3f-a70a-42db-8b99-051ea3051c84){w=480px}
```

Comment:
Thanks. As mentioned above in the reply to @hlcianfagna, I figure the rework of the page about DBeaver needs more love, so I may break it out of this PR.

Let me still ask one question: I figured the right way to use DBeaver would also be to use the [CrateDB JDBC Driver](https://cratedb.com/docs/jdbc/) instead of the [vanilla pgJDBC driver](https://jdbc.postgresql.org/), as it seems to be the right choice also for DataGrip, Apache Flink, and other integrations. Now that you are showing a settings dialogue that uses `jdbc:postgresql://`, do you think I should adjust my assumption?

NB: The tutorial [Blog: Use CrateDB With DBeaver](https://cratedb.com/blog/cratedb-dbeaver) also indicates that the CrateDB JDBC Driver should be used, including its specific protocol identifier `jdbc:crate://`.

Reply:
I've diverted concerns about DBeaver to GH-157.